spring/spring-data/src/main/java/io/quarkus/ts/spring/data/primitivetypes/BookResource.java

```diff
@@ -8,16 +8,22 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
 
 import io.quarkus.ts.spring.data.primitivetypes.model.Book;
 
 @Path("/book")
 public class BookResource {
 
     private final BookRepository bookRepository;
+    private final BookStore paged;
```

Comment:
```suggestion
    private final BookStore bookStore;
```
It's one-time use, but still, I think the semantic name looks better.